static/app/views/onboarding/integrationSetup.tsx

```diff
@@ -187,8 +173,55 @@ function IntegrationSetup(props: Props) {
     );
   };
 
+  if (isLoading) {
+    return <LoadingIndicator />;
+  }
+
+  if (!docsConfig || !dsn || !projectKeyId) {
+    return (
+      <LoadingError
+        message={t(
+          'The getting started documentation for this platform is currently unavailable.'
+        )}
+        onRetry={refetch}
+      />
+    );
+  }
+
+  const docParams: DocsParams<any> = {
+    api,
+    projectKeyId,
+    dsn,
+    organization,
+    platformKey: platform.id,
+    projectId: project.id,
+    projectSlug: project.slug,
+    isFeedbackSelected: false,
+    isPerformanceSelected: false,
+    isProfilingSelected: false,
+    isReplaySelected: false,
+    isSelfHosted,
+    platformOptions,
+    sourcePackageRegistries: {
+      isLoading: false,
+      data: undefined,
+    },
+    urlPrefix,
+  };
+
   return (
     <Fragment>
+      <SetupIntroduction
+        stepHeaderText={t('Automatically instrument %s SDK', platform.name)}
+        platform={platform.id}
+      />
+      {docsConfig.platformOptions && (
+        <PlatformOptionsControl
+          platformOptions={docsConfig.platformOptions}
+          onChange={docsConfig.onboarding.onPlatformOptionsChange?.(docParams)}
+        />
+      )}
```

Comment:
I'm not sure if we really want to display all platform options here. That can have unwanted side effects, when another option is added to the `awslambda` files.

Reply:
How about if we update the text, so that it's relevant to both auto and manual and leave the checkboxes there, but disabled to avoid the layout shift? I don't love it, but it's the only thing I could think of to avoid it.
If not, I'm also fine to merge it as-is and improve it later. As Pri pointed out, there aren't that many users who select this option.